### DISCUSSION TITLE
Clock crashes after a few days

### DISCUSSION THREAD

**Discussion by pauljcheffus** [astlouys/Pico-Green-Clock#32] [Q&A]
Hi

I’ve been running this for the last year or so and recently it seems to just stop working with random pixels displayed on the screen.

Rebooting fixes the problem for a while then it occurs again any ideas?

Cheers

Paul

**Comment by astlouys** [comment]
Hello Paul,

I never had a similar comment before and didn't see this problem on my units either.

Personally, I would first check the power supply for the presence of ripple (or temporarily replace it with another to see if it helps).

Did you try to replace the Pico to see if it is the problem ?

You said that it worked fine for the last year... Did you change any clock setting recently ?

I don't see that as a potential cause, but just for my curiosity, did you use the auto-brightness feature or was the clock set for "steady" brightness. If "steady", was it "full bright" or a little dim ?  Full bright will require more current and it the power supply is "border line", it could help giving problems...

Let me know if you find the problem. I'm curious.

André

**Reply by pauljcheffus** [reply]
Hi

No changes at all and the PSU is an Apple charger.

The clock has always been on auto dimming.

I will monitor it and see if there is a pattern to the crashing.

I don’t have another Pico currently so that’s something I will have to get.

Cheers

Paul
